### ISSUE TITLE
Fix Hostname

### ISSUE THREAD

**Issue by mike-webster** [mike-webster/spotify-views#13]
## Description
There's a HOSTNAME variable used throughout the app.  It was initially set up for the heroku domain.  I tried to update this to the custom one I bought, but it broke some things in the app.  This needs to be fixed.

**Comment by mike-webster** [comment]
I added a redirect url in the spotify app with the new hostname. I'm guessing this is the issue I was running into last time, since this would prevent the round trip needed for logging in

**Comment by mike-webster** [comment]
Added some logging - something is still broken.  The link being generated is still trying to hit https, which I changed to http because I don't have an ssl cert .

**Comment by mike-webster** [comment]
There were actually a couple of issues at play:

- When I originally updated the hostname, all I did was update the environment variable.
    - When I did that, Spotify stopped allowing the auth redirect because URL with the new hostname in it wasn't whitelisted.

After I updated the environment variable and added the new url to the whitelist, I still ran into an issue.  What this is accomplishing is updating the hostname so we're showing "spotify-views.com" instead of "spotify-views.herokuapp.com".  There's some weird issue (probably my own fault) with the CNAME/Heroku configuration where requests have to have the www prefix or they'll be rejected.

Once I updated the code to provide that along with the hostname in the url, everything works.
